src/utils/rfq_blacklist_utils.ts

```diff
@@ -0,0 +1,47 @@
+import { Connection } from 'typeorm/connection/Connection';
+
+import { BlockedAddressEntity } from '../entities/BlockedAddressEntity';
+
+/**
+ * RfqBlacklistUtils helps manage the RFQ blacklist
+ */
+export class RfqBlacklistUtils {
+    private static _instance: RfqBlacklistUtils;
+    private _blacklist: Set<string>;
+    private readonly _intervalTimeout: NodeJS.Timeout;
+
+    public static getInstance(
```

Comment:
Instead of using a timer to continuously refresh items, have we considered storing the last time we fetched the list, and simply reload whenever someone calls `isBlocked` and too much time has passed? would avoid having an other event loop running (we have like billions in the 0x API)

Reply:
give the new impl a look :D